lib/chef/knife/winrm_shared_options.rb

```diff
@@ -27,16 +27,16 @@ module WinrmSharedOptions
       # Shared command line options for knife winrm and knife wsman test
       def self.included(includer)
         includer.class_eval do
-          option :manual,
+          option :manual_list,
             :short => "-m",
             :long => "--manual-list",
             :boolean => true,
             :description => "QUERY is a space separated list of servers",
             :default => false
 
-          option :attribute,
-            :short => "-a ATTR",
-            :long => "--attribute ATTR",
+          option :winrm_attribute,
```

Comment:
needs to align with https://github.com/chef/chef/blob/master/lib/chef/knife/ssh.rb#L50